src/ray/rpc/worker/worker_client.h

```diff
@@ -0,0 +1,58 @@
+#ifndef RAY_RPC_WORKER_CLIENT_H
+#define RAY_RPC_WORKER_CLIENT_H
+
+#include <thread>
+
+#include <grpcpp/grpcpp.h>
+
+#include "ray/common/status.h"
+#include "ray/rpc/client_call.h"
+#include "ray/util/logging.h"
+#include "src/ray/protobuf/worker.grpc.pb.h"
+#include "src/ray/protobuf/worker.pb.h"
+
+namespace ray {
+namespace rpc {
+
+/// Client used for communicating with a remote worker server.
+class WorkerTaskClient {
```

Comment:
I don't think this is used.

Reply:
it will be used in NodeManager to assign task to a worker, that change will be in a later PR, but functionality wise it looks strange to only have a `server` with no `client`, so I still include it in this PR. I will add a comment.